API/API/DataAccessLayer/Repositories/IRepository.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace API.DataAccessLayer.Repositories
```

Comment:
move this to interfaces folder and define specified interface for userRepository